tools/barcode_collapse/.shed.yml

```diff
@@ -0,0 +1,21 @@
+name: barcode_collapse
+owner: rnateam
+description: "Paired End randomer aware duplicate removal algorithm"
+homepage_url: https://github.com/YeoLab/gscripts/releases/tag/1.0
+long_description: This tool is used in the eCLIP-seq Processing	Pipeline. Read in a '.bam' file where the first 9 nucleotides of the read name are the barcode and merge reads mapped to the same position that have the same barcode.
+Assumes paired end reads are adjacent to each other in the output file (i.e. only provide unsorted bams). It also assumes that there are no multimappers in the bam file, if there are multimappers behavior is undefined.
+    
+remote_repository_url: https://github.com/bgruening/galaxytools/tree/master/tools/barcode_collapse
+type: unrestricted
+categories:
+- RNA
+- Sequence Analysis
+auto_tool_repositories:
```

Comment:
If this is only one tool you don't need to define a suite and hence you can remove `auto_tool_repository` and `suite`